asset/cpu_triggered_asset_utilization.py

```diff
@@ -101,4 +81,27 @@ def loop():
     finally:
         _stop_ongoing_utilization()
 
-loop()
\ No newline at end of file
+def main(arguments=None):
+    global PARSED_ARGUMENTS
+    global UTILIZATION_IN_PROGRESS
+    global ONGOING_UTILIZATION
```

Comment:
Although not explicitly required (the global keyword will create a global variable if it doesn't exist), for documentation purposes, it is usually a best practice to also define the global variables at the top of the file, after the imports. So perhaps something like:
```
# The parsed arguments. Set in :func:`main`.
PARSED_ARGUMENTS = None
# Whether a utilization is currently in progress.
UTILIZATION_IN_PROGRESS = False
# An instance of :class:`AssetManagementUtilization` representing the current utilization, if any.
ONGOING_UTILIZATION = None
```